packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/QueryTests.kt

```diff
@@ -1280,7 +1280,7 @@ class QueryTests {
                     .asFlow()
                     .collect { /* No-op */ }
             }.let {
-                assertTrue(it.message!!.contains("Invalid numeric type"))
+                assertTrue(it.message!!.contains("Invalid property type"))
```

Comment:
It is our code. Certain things changed in how we run aggregation queries since before we could just take the value inside a `RealmValue` and see if it was a `Number` for convenience. Since `Number` can be extended there was a check for that, but it isn't necessary anymore since we work with the transport now, so it makes no sense to have the same message as before.